docs/tutorials/02_vibrational_structure.ipynb

```diff
@@ -215,59 +225,65 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "  NIIIIIII * (1268.0676746875001+0j)\n",
-      "+ INIIIIII * (3813.8767834375008+0j)\n",
-      "+ IINIIIII * (705.8633818750001+0j)\n",
-      "+ II+-IIII * (-46.025705898886045+0j)\n",
-      "+ II-+IIII * (-46.025705898886045+0j)\n",
-      "+ IIINIIII * (2120.1145593750007+0j)\n",
-      "+ IIIINIII * (238.31540750000005+0j)\n",
-      "+ IIIIINII * (728.9613775000003+0j)\n",
-      "+ IIIIIINI * (238.31540750000005+0j)\n",
-      "+ IIIIIIIN * (728.9613775000003+0j)\n",
-      "+ NINIIIII * (4.942542500000002+0j)\n",
-      "+ NI+-IIII * (-88.20174216876333+0j)\n",
-      "+ NI-+IIII * (-88.20174216876333+0j)\n",
-      "+ NIINIIII * (14.827627500000007+0j)\n",
-      "+ INNIIIII * (14.827627500000007+0j)\n",
-      "+ IN+-IIII * (-264.60522650629+0j)\n",
-      "+ IN-+IIII * (-264.60522650629+0j)\n",
-      "+ ININIIII * (44.482882500000024+0j)\n",
-      "+ NIIINIII * (-10.205891250000004+0j)\n",
-      "+ INIINIII * (-30.617673750000016+0j)\n",
-      "+ IININIII * (-4.194299375000002+0j)\n",
-      "+ II+-NIII * (42.67527310283147+0j)\n",
-      "+ II-+NIII * (42.67527310283147+0j)\n",
-      "+ IIINNIII * (-12.582898125000007+0j)\n",
-      "+ NIIIINII * (-30.61767375000002+0j)\n",
-      "+ INIIINII * (-91.85302125000007+0j)\n",
-      "+ IINIINII * (-12.582898125000007+0j)\n",
-      "+ II+-INII * (128.02581930849442+0j)\n",
-      "+ II-+INII * (128.02581930849442+0j)\n",
-      "+ IIININII * (-37.74869437500002+0j)\n",
-      "+ NIIIIINI * (-10.205891250000004+0j)\n",
-      "+ INIIIINI * (-30.617673750000016+0j)\n",
-      "+ IINIIINI * (-4.194299375000002+0j)\n",
-      "+ II+-IINI * (42.67527310283147+0j)\n",
-      "+ II-+IINI * (42.67527310283147+0j)\n",
-      "+ IIINIINI * (-12.582898125000007+0j)\n",
-      "+ IIIININI * (7.0983500000000035+0j)\n",
-      "+ IIIIINNI * (21.29505000000001+0j)\n",
-      "+ NIIIIIIN * (-30.61767375000002+0j)\n",
-      "+ INIIIIIN * (-91.85302125000007+0j)\n",
-      "+ IINIIIIN * (-12.582898125000007+0j)\n",
-      "+ II+-IIIN * (128.02581930849442+0j)\n",
-      "+ II-+IIIN * (128.02581930849442+0j)\n",
-      "+ IIINIIIN * (-37.74869437500002+0j)\n",
-      "+ IIIINIIN * (21.29505000000001+0j)\n",
-      "+ IIIIININ * (63.88515000000004+0j)\n"
+      "Vibrational Operator\n",
+      "number modes=4, number modals=[2, 2, 2, 2], number terms=50\n",
+      "  (1268.0676746875001+0j) * ( +_0_0 -_0_0 )\n",
+      "+ (3813.8767834375008+0j) * ( +_0_1 -_0_1 )\n",
+      "+ (705.8633818750002+0j) * ( +_1_0 -_1_0 )\n",
```

Comment:
Oh there was no particular reason for this. Actually our old `FermionicOp` already did it this way around, but `VibrationalOp` did not. Since we now aligned things again, this is how it turned out.
I guess we could update this slightly, but I would not consider it a blocker for the release.

Please resolve the thread, if you agree :+1:

Reply:
I wouldn't consider it a blocker either - my thought was that would read easier since the terms would align. But its something that can be looked at later.